imageroot/actions/get-configuration/20read

```diff
@@ -58,5 +61,17 @@ try:
 except:
     config['running'] = False
 
+# verify if collabora is installed on the cluster
+# propose in UI to link this nextcloud to collabora
+rdb = agent.redis_connect(privileged=False)
+config["is_collabora"] = True if rdb.get('cluster/default_instance/collabora') else False
+# retrieve all url stored in redis for all collabora instances
+array_collabora = []
+for c in [c.replace('roles/module/', '') for c in  rdb.keys('roles/module/collabora*')]:
+    url = rdb.hget('module/'+c+'/environment', 'TRAEFIK_HOST')
+    if url:
+        array_collabora.append({"name": c, "label": c+' ('+ url+')', "value": url})
```

Comment:
Do not use `keys` method because it has performance issues. The correct way to iterate over keys is `scan`.
```suggestion
for c in rdb.scan_iter('module/collabora*/environment'):
    name = c.removeprefix('module/').removesuffix('/environment')
    url = rdb.hget(c, 'TRAEFIK_HOST')
     if url:
        array_collabora.append({"name": name, "label": name+' ('+ url+')', "value": url})
```